oar/core/advisory_mgr.py

```diff
@@ -172,6 +172,22 @@ def change_advisory_status(self, status=AD_STATUS_REL_PREP):
         except Exception as e:
             raise AdvisoryException(f"change advisory status failed") from e
 
+    def get_requested_doc_security_ads(self,ads):
```

Comment:
can you add a unit test for this func

Reply:
Added, test result: 
[wewang@ovpn-12-217 tests]$   python3 -m unittest test_advisory_mgr.TestAdvisoryManager.test_get_requested_doc_security_ads
.
----------------------------------------------------------------------
Ran 1 test in 23.390s

OK

Reply:
add test_get_requested_doc_security_ads:
[wewang@ovpn-12-217 tests]$   python3 -m unittest test_advisory_mgr.TestAdvisoryManager.test_get_requested_doc_security_ads
.
----------------------------------------------------------------------
Ran 1 test in 23.390s

OK